src/test/java/io/confluent/idesidecar/restapi/cache/ClientConfiguratorStaticTest.java

```diff
@@ -262,6 +307,114 @@ record TestInput(
                 basic.auth.credentials.source=USER_INFO
                 basic.auth.user.info=%s:%s
                 """.formatted(API_KEY, API_SECRET)
+        ),
+//        new TestInput(
+//            "With mTLS for Kafka and SR",
+//            kafka,
+//            MUTAL_TLS_CREDENTIALS,
+//            schemaRegistry,
+//            MUTAL_TLS_CREDENTIALS,
+//            true,
+//            true,
+//            false,
+//            null,
+//            """
+//                bootstrap.servers=localhost:9092
+//                security.protocol=SSL
+//                ssl.truststore.location=/path/to/truststore
+//                ssl.truststore.password=%s
+//                ssl.keystore.location=/path/to/keystore
+//                ssl.keystore.password=%s
+//                ssl.key.password=%s
+//                """.formatted(MTLS_TRUSTSTORE_PASSWORD, MTLS_KEYSTORE_PASSWORD, MTLS_KEY_PASSWORD),
+//            """
+//                schema.registry.url=http://localhost:8081
+//                ssl.truststore.location=/path/to/truststore
+//                ssl.truststore.password=%s
+//                ssl.keystore.location=/path/to/keystore
+//                ssl.keystore.password=%s
+//                ssl.key.password=%s
+//                """.formatted(MTLS_TRUSTSTORE_PASSWORD, MTLS_KEYSTORE_PASSWORD, MTLS_KEY_PASSWORD)
+//        ),
```

Comment:
This should probably be removed as-is, since we don't have `MutualTLSCredentials` anymore. And we somehow need to change `TestInputs` so that we can specify the TLS config to the `ClientConfigurator` methods.

IIUC, we are never really testing with TLS or with not-verifying certs.